scripts/background.js

```diff
@@ -58,6 +58,11 @@ function handle_context_menu_click(info, tab) {
 
 var commands = (browser.commands || chrome.commands);
 if (commands) {  // TODO somehow this is undefined
+
```

Comment:
I wrote this comment here to let you know that in the `manifest.json` there was missing the actual command `run-lext` which you were expecing here.

Reply:
Thanks! I realized the problem was in the calling of the `main` function which was underneath this... I restructured this quite a bit now and replaced the main function, now the shortcut works! :)